core/models.py

```diff
@@ -1,3 +1,5 @@
 from django.db import models
 
-# Create your models here.
+# Create your models here..
```

Comment:
Remove this comment

Reply:
That is cool bro i will do that in a minute